src/sentry/integrations/api/endpoints/organization_coding_agents.py

```diff
@@ -82,3 +172,279 @@ def get(self, request: Request, organization) -> Response:
+                    repository=repo,
+                    branch_name=sanitize_branch_name(autofix_state.request["issue"]["title"]),
+                )
+
+                # Launch the agent
+                coding_agent_state = installation.launch(launch_request)
+
+                # Store the coding agent state to Seer
+                repo_store_success = store_coding_agent_state_to_seer(
+                    run_id=run_id,
+                    coding_agent_state=coding_agent_state,
+                )
+
+                if not repo_store_success:
+                    logger.warning(
+                        "coding_agent.seer_store_failed",
+                        extra={
+                            "organization_id": organization.id,
+                            "run_id": run_id,
+                            "repo_name": repo_name,
+                        },
+                    )
+
+                results.append(
+                    {
+                        "repo_name": repo_name,
+                        "coding_agent_state": coding_agent_state,
+                    }
+                )
+
+            except Exception as e:
+                logger.exception(
+                    "coding_agent.repo_launch_error",
+                    extra={
+                        "organization_id": organization.id,
+                        "run_id": run_id,
+                        "repo_name": repo_name,
+                        "error": str(e),
+                    },
+                )
+                # Continue with other repos instead of failing entirely
+                continue
+
+        return results
+
+    def post(self, request: Request, organization) -> Response:
+        """Launch a coding agent."""
+        if not features.has("organizations:seer-coding-agent-integrations", organization):
+            return Response({"detail": "Feature not available"}, status=404)
+
+        try:
+            # Validate request and get integration
+            validation_result = self._validate_and_get_integration(request, organization)
+            if isinstance(validation_result, Response):
+                return validation_result
+            integration, installation = validation_result
+
+            # Get autofix state
+            run_id = request.data.get("run_id")
+            if run_id is None:
+                return Response({"error": "run_id is required"}, status=400)
+
+            try:
+                run_id = int(run_id)
+            except (ValueError, TypeError):
+                return Response({"error": "Invalid run_id"}, status=400)
+
+            autofix_state = self._get_autofix_state(run_id, organization)
+            if autofix_state is None:
+                return Response({"error": "Autofix state not found"}, status=400)
+
+            # Get and validate trigger_source
+            trigger_source = request.data.get("trigger_source", "solution")
+            if trigger_source not in ["root_cause", "solution"]:
```

Comment:
i've seen these strings in a few places, can we make a `StrEnum` that we can use across the codebase? in order to avoid [magic strings](https://en.wikipedia.org/wiki/Magic_string)